index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Modern Javascript Developer Mastery Exercise</title>
+	<!-- Latest compiled and minified CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+	<link rel="stylesheet" type="text/css" href="robots.css">
+</head>
+<body>
+
+<h1 id="title">Robot Battledome</h1>
+
+	<div id="first-robot" class="robots col-md-4">
+		<label>Enter Robot Name:</label>
+		<input type="text" id="robot1-name" class="name-input">
+
+		<label>Select Robot Type:</label>
+		<select id="robot1-type">
```

Comment:
once this is selected and created, remove the selection option (and name input field). I can create a new robot in the middle of the battle.